src/build-scripts/docker-centos.bash

```diff
@@ -0,0 +1,123 @@
+# docker run -ti --rm -v $(pwd):/jc quay.io/pypa/manylinux2014_x86_64:latest bash
+set -ex
+
+if [[ "$OSTYPE" != "msys" ]]; then
+    yum install ninja-build wget less ccache -y
```

Comment:
If we are going through `scikit-build-core`, we shouldn't have to install ninja.